### ISSUE TITLE
Skipping songs without touching the next button

### ISSUE THREAD

**Issue by angelovseidel** [OuterTune/OuterTune#181]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/dd3boh/OuterTune/actions).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

1. Pick a song
2. Then this will appear Playback error: source error (2004): Response code:403
3. The next song will automatically play. 

### Expected behavior

This should only happen when the next button is pressed. 

### Actual behavior

This happened without hitting the next button. 

### Screenshots/Screen recordings

https://github.com/user-attachments/assets/8dab8241-c129-4976-acb1-c39d2bdbe73e

### Logs

This issue is easy to reproduce. 

### OuterTune version

0.6.4

### Device & Android version

Honor phone, Android 14

**Comment by ghost** [comment]
Bro just a question does OuterTune for you force quit after the error code shows in the app?

**Comment by angelovseidel** [comment]
> Bro just a question does OuterTune for you force quit after the error code shows in the app?

No, it does not. Is your yt account connected? Maybe that's the reason. Yt has some problems now. 🫠

**Comment by ghost** [comment]
> No, it does not. Is your yt account connected? Maybe that's the reason. Yt has some problems now. 🫠

Yes, My YT account is connected. But why is mine OuterTune kept force quitting. Tried uninstall/reinstall but the problem persist... Maybe I will just wait for the next update 🥲

**Comment by mattcarter11** [comment]
@angelovseidel this is no bug, currently outer tune skips a song when it fails. If it happens two times on a row it then pauses.

I've made a PR where in the next release you will be able to control if it should pause, skip or wait on error.

So, if this answers and solves the problem, please close the bug.
